c8/fabric.py

```diff
@@ -147,6 +148,15 @@ def redis(self):
         """
         return RedisInterface(self._conn, self._executor)
 
+    @property
+    def plan(self):
```

Comment:
plan APIs don't have a dependency on fabric so maybe we can add the wrapper in client itself?